Python/pywarpx/picmi.py

```diff
@@ -1908,6 +1908,32 @@ def set_write_dir(self):
             file_prefix = (self.file_prefix or self.name)
             self.diagnostic.file_prefix = os.path.join(write_dir, file_prefix)
 
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class ParticleFieldDiagnostic(object):
+    """
+    Parameters
+    ----------
+    name: str
+        Name of particle field diagnostic. If a component of a vector field, the coordinate
```

Comment:
For this comment, perhaps be more specific.
`If a component of a vector field, for the openPMD viewer to to treat it as a vector, the coordinate (i.e. x, y, z) should be the last character.`